okio/js/src/main/kotlin/okio/Sink.kt

```diff
@@ -15,4 +15,12 @@
  */
 package okio
 
-actual interface Sink
+actual interface Sink {
```

Comment:
can we just put these interfaces in common?

Reply:
I don't think so, b/c of the java.io deps.

Reply:
Right, we're adding supertypes for `Sink` on the JVM side and need an `actual` for this.